packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -722,19 +722,14 @@ function doRemovePoint(
         return state;
     }
 
-    let nextFocusedPointIndex: number | null;
-    if (state.interactionMode === "mouse") {
```

Comment:
I confirmed with Design that we still want the focus to move to the next item for mouses as well.